### PR TITLE
[KOGITO-240] - include s2i and some behave tests for the Kogito images

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]
Nice work

File: s2i/tests/features/kogito-quarkus-index.feature [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@quay.io/kiegroup/kogito-data-index
+
+Feature: Kogito-quarkus-ubi8 feature.
```
Comment (ricardozanini) [inline s2i/tests/features/kogito-quarkus-index.feature:3]: Could you please add a new Scenario with those values for env vars? Might be an error in the image that I'd like to check. Plus these values are closer to a real use case.

```
INFINISPAN_CREDENTIAL_SECRET=infinispan-credentials
INFINISPAN_AUTHREALM=default
INFINISPAN_SASLMECHANISM=PLAIN
QUARKUS_INFINISPAN_CLIENT_SERVER_LIST=server:11222
MP_MESSAGING_INCOMING_KOGITO_PROCESSINSTANCES_EVENTS_BOOTSTRAP_SERVERS=mycluster:9092
KOGITO_PROTOBUF_FOLDER=/home/kogito
KOGITO_PROTOBUF_WATCH=true
INFINISPAN_USEAUTH=true
```

**Review by ricardozanini — CHANGES_REQUESTED** [review]
Could you kindly add this new scenario?

File: s2i/tests/features/kogito-quarkus-index.feature (hunk 1 above)
Comment (spolti) [reply s2i/tests/features/kogito-quarkus-index.feature:3]: added.

**Review by ricardozanini — APPROVED** [review]
LGTM

**Pull request merged**